### DISCUSSION TITLE
iterative suews runs to avoid memory errors

### DISCUSSION THREAD

**Discussion by DimitrisTsi** [UMEP-dev/UMEP#677] [Ideas]
The following code has been developed to avoid memory errors that crash SUEWS when running large grids for long time periods (e.g. full year).

[suews_iterative_runs.txt](https://github.com/user-attachments/files/17977432/suews_iterative_runs.txt)

**Comment by biglimp** [comment]
Great! We will implement it into UMEP in upcoming versions. Other users can also use the code directly in SuPy.
